pallets/xcmp-handler/src/tests.rs

```diff
@@ -157,14 +158,14 @@ fn returns_fee() {
 }
 
 #[test]
-fn errors_if_not_overflow() {
```

Comment:
i'm fine with us changing the test, but just wanna make sure it's not unintentional. do we still need a case where xcm fee overflows somehow?

Reply:
that still causes an overflow

Reply:
ahh sorry sorry, i misread test name. ignore me